core/app/models/spree/tax/output/item.rb

```diff
@@ -0,0 +1,10 @@
+module Spree
+  module Tax
+    module Output
+      class Item
+        include ActiveModel::Model
+        attr_accessor :id, :tax_rate, :amount, :included_in_price
```

Comment:
1. Love the idea of moving labels to the output object. I'll put together that change in a few.
2. We need some way of tying the rate to the line item or shipment it should be applied to. The ID is our only real way of doing so. Sorting both lists won't always work. (For example, some items can be tax exempt and shouldn't have any rates associated to them.)
3. As you've commented later, this is due to adjustments requiring a source.